frontend/src/components/job/List.tsx

```diff
@@ -29,6 +29,24 @@ import ResourceListView from '../common/Resource/ResourceListView';
 import { SimpleTableProps } from '../common/SimpleTable';
 import LightTooltip from '../common/Tooltip/TooltipLight';
 
+export function makeJobStatusValue(job: Job) {
+  if (!job?.status?.conditions) {
+    return 'Unknown';
```

Comment:
Please make this string translatable since it's not coming from Kubernetes' statuses.

Reply:
i also used dash as it also undefined